Zadatak1.ps1

```diff
@@ -0,0 +1,59 @@
+﻿param(
+    [Parameter(Mandatory=$true)]
+    [string]$dateFolder # Datum kao parametar u forrmatu 'yyyy-MM-dd'
+)
+
+$rootDir = "izvodi"
+
+# Osigurati da izvodi postoje
+if (-Not (Test-Path $rootDir)) {
+    Write-Error "'$rootDir' direktorijum ne postoji."
+    exit
+}
+
+# Prolazimo kroz svaki petocifreni folder
+Get-ChildItem -Path $rootDir -Directory | Where-Object { $_.Name -match '^\d{5}$' } | ForEach-Object {
+    $folder = $_
+    $dateFolderPath = Join-Path -Path $folder.FullName -ChildPath $dateFolder
+    
+    # Nastavlja ako trazeni datum folder postoji
+    if (Test-Path $dateFolderPath) {
+        # Privremeni folder za ekstraktovane fajlove
+        $tempExtractionPath = Join-Path -Path $dateFolderPath -ChildPath "temp_extraction"
+        New-Item -Path $tempExtractionPath -ItemType Directory -Force | Out-Null
```

Comment:
Creating a temporary folder like this is prone to errors. If the script is halted for whatever reason (error, manually, etc.) this folder will stay. Worst, if the content of the original directory is changed between the runs, this folder will contain remains of previous extraction which are no longer present and thus will get packed into the new archive